### DISCUSSION TITLE
Daily Borg Backup not working

### DISCUSSION THREAD

**Discussion by KaeTuuN** [nextcloud/all-in-one#3985] [Questions] [answered]
### Steps to reproduce
1. Open AIO Page
2. Set Daily Backup to 01:00 UTC
3. Wait 3 days
4. Nothing happens

### Expected behavior 
AIO does a Backup every night at 01:00 UTC

### Actual behavior 
Nothing happens at 01:00 UTC

### Host OS 
Debian 12

#### Nextcloud AIO version 
7.9.0
At least also happened with: 7.8.1

#### Current channel 
latest

#### Other valuable info 
Backup itself works if I manually start it, or if AIO makes an Update/Upgrade.

**Comment by joshtrichards** [comment]
If you delete the scheduled backup time then - after it refreshes - enter a new backup time in does it resolve it?

Not saying it should, but trying to figure out why you're encountering this.

Any additional backup directories configured?

Anything interesting in the Docker logs?

**Comment by KaeTuuN** [comment, marked as answer]
> If you delete the scheduled backup time then - after it refreshes - enter a new backup time in does it resolve it?

Seemed to resolve my problem (forgot to unmount the repo yesterday so the logs said "Failed to create/acquire the lock...". But at least the Container is starting!


> Any additional backup directories configured?

No.


> Anything interesting in the Docker logs?

Nothing in the Mastercontainer nor the Borg Container. It seems as if the settings have not been persisted the first time.

**Comment by KaeTuuN** [comment]
Works now! Thanks for your help.
When I find some time I'll try to reproduce this issue.

@joshtrichards You can decide, if you want to leave this issue open for that.
